.github/workflows/healthchecks_dapr_ci.yml

```
@@ -0,0 +1,65 @@
+name: HealthChecks Dapr CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+    paths:
+      - src/HealthChecks.Dapr/**
+      - test/HealthChecks.Dapr.Tests/**
+      - test/_SHARED/**
+      - .github/workflows/healthchecks_Dapr_ci.yml
```

```suggestion
      - .github/workflows/healthchecks_dapr_ci.yml
```

Important for Linux.